pkg/updates/file_test.go

```diff
@@ -558,15 +558,180 @@ func GraphEqual(t testing.TB, a, b *UpdateGraph) {
 		for _, bc := range b.Channels {
 			if ac.EqualIdentity(bc) {
 				equalCount++
-				require.ElementsMatch(t, ac.Nodes, bc.Nodes)
-				for source, edges := range ac.Edges {
-					require.ElementsMatch(t, edges, bc.Edges[source])
-				}
-				for source, edges := range bc.Edges {
-					require.ElementsMatch(t, edges, ac.Edges[source])
-				}
+				ChannelEqual(t, ac, bc)
 			}
 		}
 	}
 	require.Equal(t, len(a.Channels), equalCount)
 }
+
+func ChannelEqual(t testing.TB, ac, bc Channel) {
+	require.ElementsMatch(t, ac.Nodes, bc.Nodes)
```

Comment:
unsure  about this one since you may actually want the line where it failed since there are 2 assertions going on
```suggestion
        t.Helper()

	require.ElementsMatch(t, ac.Nodes, bc.Nodes)
```